posts/urls.py

```diff
@@ -6,4 +6,11 @@
 urlpatterns=[
     path('', view=views.post_list, name='post_list'),
     path('create/', view=views.post_create, name='post_create'),
-]
\ No newline at end of file
+    path('<int:pk>/', view=views.post_detail, name='detail'),
+    path('like/<int:pk>/', view=views.handle_like, name='like'),
```

Comment:
You should update the URLs. For example, `posts/like/<post_id>/` should be `posts/<post_id>/like/`